src/server.c

```diff
@@ -1735,6 +1731,10 @@ void afterSleep(struct aeEventLoop *eventLoop) {
     /* Update the time cache. */
     updateCachedTime(1);
 
+    /* Update command time snapshot in case it'll be required without a command
+     * e.g. somehow used by module timers. */
+    server.cmd_time_snapshot = server.mstime;
```

Comment:
should put inside the `ProcessingEventsWhileBlocked` check below?

Reply:
ohh, i thought i looked into that and concluded that that but i now see i missed it.

i also see we have a comment saying `Do NOT add anything above moduleAcquireGIL !!!` and we do have code above it :disappointed:

Reply:
i see that #10300 violated the above mentioned GIL comment, but luckily it wasn't released yet, so i'm fixing it in this PR (noting that in the top comment).
also removed some blank lines, hoping it'll reduce the chance we'll push something before this warning again.
@enjoy-binbin FYI.

Reply:
oops. sorry for that. i don't recall any of this.. i must not have read the comment carefully. thanks for the fix